README.md

```diff
@@ -1,6 +1,66 @@
 Support for detecting and being notified when an object is mutated.
 
+An observable, simply put, is a continuous stream of events over time.
```

Comment:
How about:

An `Observable` is a way to be notified of a continuous stream of events over time.